services/inventory/services.go

```diff
@@ -224,6 +224,32 @@ func (ss *ServicesService) AddPostgreSQL(ctx context.Context, params *models.Add
 	return res.(*inventorypb.PostgreSQLService), nil
 }
 
+// AddProxySQL inserts ProxySQL Service with given parameters.
+//nolint:dupl,unparam
+func (ss *ServicesService) AddProxySQL(ctx context.Context, params *models.AddDBMSServiceParams) (*inventorypb.ProxySQLService, error) {
+	// TODO Decide about validation. https://jira.percona.com/browse/PMM-1416
+	// Both address and socket can't be empty, etc.
```

Comment:
Does not apply – we don't have `socket`, and we decided about validation (at least for now).